src/components/PageLayout/PageLayout.tsx

```diff
@@ -39,24 +38,18 @@ export const PageLayout: FC<PageLayoutProps> = ({contentWidth = 2, title = '', c
       </Head>
       <PageTitleContainer.Provider initialState={title}>
         <BannerContainer.Provider>
-          <div className={styles.pageContainer}>
+          <Stack sx={{minHeight: '100vh', position: 'relative'}}>
             <TopGrid />
-            <MenuMain />
-            <StromLogo />
-            <div className={styles.grid}>
-              <div
-                className={clsx(
-                  styles.mainContent,
-                  contentWidth === 1 && styles.col1,
-                  contentWidth === 2 && styles.col2,
-                  contentWidth === 3 && styles.col3,
-                )}
-              >
+            <Grid container sx={{height: '100%', flex: 1}} columns={{xs: 6, md: 12}}>
+              <Grid md={3} sx={{position: 'relative', display: {xs: 'none', md: 'block'}}}>
+                <StromLogo />
+              </Grid>
+              <Grid xs={6} md={contentWidth * 3} sx={{pb: 2, px: 1}}>
```

Comment:
pride mi na prvu kus zvlastne, ze sa upravuje aj pocet stlpcov hore, aj sirka elementov v pocte stlpcov, co zaberaju... mozno by som dal:
- parent - 12 stlpcov (takze asi staci zmazat cely `columns` prop?)
  - strom - `xs={3}`, aj ked xs je schovane, je to takto menej matuce. alebo este skusit `xs={0}`, ci ide 😄 
  - zvysok - `xs={12} md={contentWidth * 3}`

Reply:
tak defaultna hodnota je, ze tie bez udanej hodnoty si to zoberu a rovnomerne rozdelia zvysok riadku

xs={0} je asi cajk :-D